specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/2018-01-01/ResourceHealth.json

```diff
@@ -0,0 +1,876 @@
+{
```

Comment:
In accordance with the new format, introduced late December 2017, please update this PR so that the "2018-01-01" directory is in either a folder named "stable" or "preview". e.g.

```
resource-manager/Microsoft.ResourceHealth/2018-01-01/ResourceHealth.json -> resource-manager/Microsoft.ResourceHealth/preview/2018-01-01/ResourceHealth.json
```

or 

```
resource-manager/Microsoft.ResourceHealth/2018-01-01/ResourceHealth.json -> resource-manager/Microsoft.ResourceHealth/preview/2018-01-01/ResourceHealth.json
```